Tools/ros2/ardupilot_dds_tests/test/ardupilot_dds_tests/test_geopose_msg_received.py

```diff
@@ -41,7 +47,7 @@
 CMAC_HEADING = 353
 
 
-def ros_quat_to_heading(quat):
+def ros_quat_to_heading_deg(quat):
     # By default, scipy follows scalar-last order – (x, y, z, w)
```

Comment:
`scipy` is a large library to pull in just for quaternions and rotations. Have you thought about `transforms3d` (https://matthew-brett.github.io/transforms3d/)?

Reply:
Yea, I just saw scipy was already used a bunch of places, including in the setup script, and thought the team would prefer a library they are familiar with.
https://github.com/ArduPilot/ardupilot/blob/72a013952c62cf645c8f668620ba52a41debe042/Tools/environment_install/install-prereqs-ubuntu.sh#L195

Your link looks like a much more focused library, but it's not available on `apt` yet. 
https://github.com/matthew-brett/transforms3d/issues/65